parttwo.asm

```diff
@@ -0,0 +1,86 @@
+.data
+
+intro:	.asciiz "Hello. I am Nick Gordon. This program requests two integers.\nIt then does three calculations, and saves them to data.\nIt then prints those results.\n"
+outro:	.asciiz "This was the program! Hope you have a good day!"
+prompt1:	.asciiz "Please enter integer one: "
+prompt2:	.asciiz "Please enter integer two: "
+space:	.asciiz "\n" # because I have to use an entire word to store the newline :(
+sum:		.word	0
+sumstring:	.asciiz "Sum: "
+diff:		.word	0
+diffstring:	.asciiz "Diff: "
+prod:		.word	0
+prodstring: .asciiz "Prod: "
+
+.text
+
+la $s5, sum
+la $s6, diff
+la $s7, prod
+
+la $a0, intro		# announces the introduction and purpose
+li $v0, 4
+syscall
+
+la $a0, prompt1		# prints prompt for the first integer
+li $v0, 4
+syscall
+li $v0, 5			# syscall to request the integer
+syscall
+add $s0, $s0, $v0		# placing the integer in register
```

Comment:
-2 Lines 30 and 37 are inherently dangerous
If you want to move from $v0 to $s0 or $s1, consider using:
	add $s0, $zero, $v0
instead of:
	add $s0, $s0, $v0
It works in this case, but if this code were part of a loop or a larger program it would break because you would be adding whatever happened to be in that register at the time.